.github/workflows/production_actions.yml

```
@@ -25,6 +25,11 @@ jobs:
       with:
         dotnet-version: 8.0.x
 
+    - name: Setup .NET 10.0
+      uses: actions/setup-dotnet@v1
```

actionlint (1.7.12)

[error] the runner of "actions/setup-dotnet@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)